compiler/test/dotc/pos-test-pickling.blacklist

```diff
@@ -19,6 +19,7 @@ i6507b.scala
 i12299a.scala
 i13871.scala
 i15181.scala
+i15922.scala
```

Comment:
How come?

Reply:
I did not have the time to follow up on this.